### PR TITLE
aliceincradle: Add version 0.24g

### PR DESCRIPTION
**Pull request by lwd-temp**
- [x] I have read the [Contributing Guide](https://github.com/Calinou/scoop-games/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by lwd-temp** [comment]
Note: There's only `checkver`, but no `autoupdate` because its download link is provided directly in HTML, not the API. And Scoop does not support find download link by regex other than `checkver` URL.

File: bucket/aliceincradle.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "version": "0.24g",
+  "description": "A soft Ryona 2D action game in which MP is thoroughly absorbed when the heroine is defeated.",
```
Comment (Calinou) [inline bucket/aliceincradle.json:3]:
```suggestion
  "description": "A soft Ryona 2D action game in which MP is thoroughly absorbed when the heroine is defeated",
```

**Comment by Calinou** [comment]
Thanks for opening a pull request!

Unfortunately, given the 18+ nature of the game, I can't add it to scoop-games for legal reasons (we don't ask all users of scoop-games whether they're over 18 when they enable the bucket). To avoid any risks, I'll have to reject this pull request.

**Comment by lwd-temp** [comment]
Understand.

Will ask users if they're over 18 in a preinstall script acceptable?

**Pull request closed without merging**